src/test/unit/scheduler/test_analysis.py

```diff
@@ -109,6 +120,20 @@ def test_process_next_analysis_unknown_plugin(self):
             self.sched._process_next_analysis_task(test_fw)
             assert not spy.was_called(), 'unknown plugin should simply be skipped'
 
+    # TODO this marker should not completely overwrite the outer markers
+    @pytest.mark.cfg_defaults(
+        {
+            'file_hashes': {
+                'hashes': 'md5, sha1, sha256, sha512, ripemd160, whirlpool',
+            },
+            'printable_strings': {
+                'min-length': 6,
+            },
+            'dummy_plugin_for_testing_only': {
+                'mime_whitelist': 'foo, bar',
+            },
+        }
+    )
```

Comment:
I don't know if there is a good solution to this
the easiest obvious solution though: saving the common parts in a variable
```suggestion
    # TODO this marker should not completely overwrite the outer markers
    @pytest.mark.cfg_defaults(
        {
            **common_stuff,
            'dummy_plugin_for_testing_only': {
                'mime_whitelist': 'foo, bar',
            },
        }
    )
```

Reply:
The test was to resolve the TODO in conftest.py.
Now the markers are combined.